pkg/catalogv2/helmop/operation.go

```diff
@@ -710,6 +709,9 @@ func (s *Operations) getChartCommand(namespace, name, chartName, chartVersion st
 		return Command{}, err
 	}
 
+	// secret key names doesn't accept strings with '/' character
+	chartName = strings.Replace(chartName, "/", "-", -1)
```

Comment:
Can we introduce a validation to enforce invalid names instead?

Reply:
@pjbgf @mattfarina This has nothing to do with the chart name. The variable name was poorly created. This variable is the name of the file to be created `values-%s.yaml` and chart tar file. I have updated the codebase to have a sanitize function according to the pattern of the current codebase. There is already a sanitize function in the file.

 Let me know if you have any more questions.